Test/Transaction.juvix

```diff
@@ -86,3 +93,20 @@ checkPartitionResources
           | member? r resourceSet && sigCheck := r :: acc
           | else := acc;
   in resourceCommitments, resourceNullifiers;
+
+verify {expectedBalance : Balance := zeroBalance} (tx : Transaction) (ctx : Context) : Context :=
```

Comment:
Note that I had to put the `expectedBalance` default argument in the beginning so that the partial function application syntax in `tests/Main.juvix` still works.
I normally would have put it in the end, but the use of the `|>` operator later on forced me to put it as the first argument (maybe because I am lacking the ability to do it otherwise).
I think this is an interesting observation and openness for change should be considered when designing APIs like this.